src/itouch.cpp

```diff
@@ -1,8 +1,8 @@
-#include <irods/rodsClient.h>
-#include <irods/client_connection.hpp>
-#include <irods/irods_version.h>
-#include <irods/touch.h>
-#include <irods/irods_exception.hpp>
+#include "rodsClient.h"
```

Comment:
@korydraughn do you know some reason for the angle brackets?

Reply:
oh i see the commit message... the non-default search paths...

Reply:
Yeah, `#include <irods/*>` will only work if all the headers are in a folder named `irods` that's in a system include search path. The include path we pass to the compiler already has the `irods` bit, so it would be skipped over while searching (assuming it does count as a system include search path).